mnefun/_mnefun.py

```diff
@@ -186,6 +187,32 @@ class Params(Frozen):
         High-pass transition band.
     movecomp : str | None
         Movement compensation to use. Can be 'inter' or None.
+    sss_type : str
+        signal space separation method. Must be either 'maxfilter' or 'python'
+    int_order : int
+        Order of internal component of spherical expansion.
+    ext_order : int
+        Order of external component of spherical expansion.
+    tsss_dur : float | None
+        Apply spatiotemporal SSS with specified buffer duration
+        (in seconds). Elekta's default is 10.0 seconds in MaxFilter™ v2.2.
+        Spatiotemporal SSS acts as implicitly as a high-pass filter where the
+        cut-off frequency is 1/st_dur Hz. For this (and other) reasons, longer
+        buffers are generally better as long as your system can handle the
+        higher memory usage. To ensure that each window is processed
+        identically, choose a buffer length that divides evenly into your data.
+        Any data at the trailing edge that doesn't fit evenly into a whole
+        buffer window will be lumped into the previous buffer.
+    st_correlation : float
+        Correlation limit between inner and outer subspaces used to reject
+        ovwrlapping intersecting inner/outer signals during spatiotemporal SSS.
+    trans_to : str | None
+        The destination location for the head. Can be ``None``, which
+        will not change the head position, or a string path to a FIF file
+        containing a MEG device<->head transformation.
```

Comment:
Again I'm not sure this is how we want to do this. There were other options that existed already and we should try to unify our treatment of the MaxFilter and `maxwell_filter` operations. Can you try to make it so as many options as possible apply to both versions of the algorithms? We'll need to expand `run_sss.sh` to accept a few more options.